packages/keystatic/src/component-blocks/cloud-image-preview.tsx

```diff
@@ -10,65 +10,189 @@ import { Icon } from '@keystar/ui/icon';
 import { externalLinkIcon } from '@keystar/ui/icon/icons/externalLinkIcon';
 import { useConfig } from '../app/shell/context';
 
-export function CloudImagePreview(
-  props: PreviewProps<
-    ObjectField<typeof import('./cloud-image-schema').cloudImageSchema>
-  > & {
-    onRemove(): void;
-  }
-) {
-  const src = props.fields.src.value;
-  const alt = props.fields.alt.value;
+import { detect } from 'detect-browser';
```

Comment:
We include `@react-aria/utils`, you should be able to get this behaviour from the [platform](https://github.com/adobe/react-spectrum/blob/main/packages/%40react-aria/utils/src/platform.ts) utils.